modules/ocf_docker/files/garbage-collect

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+# Always make sure the registry is started, no matter what fails.
+trap 'systemctl start docker-registry' EXIT
+
+# Delete all images older than 2 months.
+deckschrubber -repos 9999 -month 2
+
+systemctl stop docker-registry
+docker run --rm \
+    -v /var/lib/registry:/var/lib/registry \
+    registry:2 \
```

Comment:
This is not a comment on this PR as much as a general one, but I do wonder how we should manage versioning our running third-party docker images, I mean we just pin this to `2` but it won't pull any new versions unless it's shut down, the local cache is cleaned, and it's started up again. This could be done automatically, but then it has the potential to break on some unexpected upgrade change. It could be pinned to a more specific version like `2.7`, but then it needs to be bumped like [marathon-lb](https://github.com/ocf/puppet/blob/487324734f4d34d5ae68cb79918ac0455cddc1d7/modules/ocf_mesos/files/master/load_balancer/ocf-lb.service#L7) does. Altogether, I'm not sure what's the best path here.

(Yes, this is doing the garbage collection part, not actually managing the service, but I thought I'd dump this thought somewhere)

Reply:
Pinning it to a more specific version would probably be better. I was thinking maybe setting an environment variable in biohazard using hiera to control which image version is used, and then bumping that hiera variable. But maybe that's unnecessary, and the versions can just be changed in the script and the systemd unit file.